index.html

```diff
@@ -435,7 +435,70 @@ <h4 class="languages">Skills</h4>
 
         </section>
 
-        
+        <section class="contact-me">
+            <h3 class="intro">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h3>
+
+            <form id="contact-form" action="https://formspree.io/f/mrgjoqgg" method="post">
+                <ul>
+                    <li>
+                        
+                        <input type="text" id="text" placeholder="Full Name" name="User_name" maxlength="30" required>
+                    </li>
+
+                    <li>
+                       <input type="email" id="email" placeholder="E-mail address" name="e-mail" required>
+                    </li>
+                    <li>
+                    
+                     <textarea id="more-info" name="more-info" placeholder="Enter your text here" maxlength="500" required></textarea>
+                    </li>
+                </ul>
+
+                <button class="button resume project-button" type="submit" >Get in touch</button>
+                
+            
+            </form>
```

Comment:
- [ ] Hi, a nice implementation of the contact form and design but your form is not responsive to mobile view.
Kindly make your form responsive to all screen size
![matchres](https://user-images.githubusercontent.com/55596970/165404256-3e510633-3705-4b57-adc8-d7f6a33cb948.png)
.

Reply:
The webpage has been optimized to become responsive.